rust/src/connection/runtime.rs

```diff
@@ -47,7 +47,7 @@ impl BackgroundRuntime {
     pub(crate) fn new() -> Result<Self> {
         let is_open = AtomicCell::new(true);
         let (shutdown_sink, mut shutdown_source) = unbounded_async();
-        let async_runtime = runtime::Builder::new_current_thread().enable_time().enable_io().build()?;
+        let async_runtime = runtime::Builder::new_multi_thread().enable_time().enable_io().build()?;
```

Comment:
@dmitrii-ubskii  do you think this is needed?

Reply:
No. We're creating a new blocking thread, the number of runtime threads doesn't need to change.